spec/lob_spec.rb

```diff
@@ -2,77 +2,44 @@
 
 describe Lob do
 
-  after do
-    Lob.api_key = nil
-    Lob.api_version = nil
-  end
-
-  it "should return the resource object for the valid version" do
-    Lob(api_key: "test", api_version: "2014-11-25").must_be_kind_of(Lob::Client)
-  end
-
-  it "should *not* raise an error if API key has been on module and not passed as option" do
-    Lob.api_key = "test"
-    Lob().wont_be_nil
+  it "should return the client object for the valid version" do
+    @lob = Lob::Client.new(api_key: "test", api_version: "2014-11-25")
+    @lob.must_be_kind_of(Lob::Client)
+    assert @lob.config[:api_version] == "2014-11-25"
   end
 
   it "should pass the API key to the resource for the version" do
-    Lob(api_key: "test").options[:api_key].must_equal "test"
+    Lob::Client.new(api_key: "test").config[:api_key].must_equal "test"
   end
 
   it "should raise an error if no API key is supplied" do
     assert_raises ArgumentError do
-      Lob()
-    end
-  end
-
-  it "should allow detailed configuration" do
-    Lob.configure do |config|
-      config.api_key = "test"
-      config.api_version = "2014-11-24"
-      config.protocol = "https"
-      config.api_host = "api.lob.com"
+      Lob::Client.new()
     end
-
-    Lob.api_key.must_equal "test"
-    Lob.api_version.must_equal "2014-11-24"
-    Lob.protocol.must_equal "https"
-    Lob.api_host.must_equal "api.lob.com"
-  end
-
-  it "should work with Lob.load" do
-    Lob.load(api_key: "test").options[:api_key].must_equal "test"
-
-    Lob.api_key = "test"
-    Lob.load.wont_be_nil
-
-    Lob.load(api_key: "test").must_be_kind_of(Lob::Client)
   end
 
   it "should handle API errors gracefully" do
-    lob = Lob.load(api_key: API_KEY)
+    @lob = Lob::Client.new(api_key: API_KEY)
```

Comment:
nit: Does this have to be an instance variable? (Same for other `@lob`s in this file).